site/content/blog/markdowndb-launch.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Announcing the official launch of MarkdownDB! 🚀
+description: We're excited to announce the official launch of MarkdownDB, an open library to transform markdown content into SQLite database. And now, it has a dedicated website at markdowndb.com
+date: 2023-10-11
+authors: ['Ola Rubaj']
+filetype: 'blog'
+---
+
+Hello, dear readers!
+
+We're excited to announce the official launch of MarkdownDB, an open library to transform markdown content into SQLite database. And now, it has a dedicated website!
+
+Visit https://markdowndb.com/
+
+## 🔍 What is MarkdownDB?
+
+MarkdownDB transforms your Markdown content into a queryable, lightweight SQLite database. Imagine being able to treat your collection of markdown files like entries in a database! Ever thought about fetching documents with specific tags or created in the past week? Or, say, pulling up all tasks across documents marked with the "⏭️" emoji? With MarkdownDB, all of this (and more) is not just possible, but also a breeze.
+
+## 🌟 Features
+
+- Rich metadata extracted including frontmatter, links and more.
+- Lightweight and fast indexing 1000s of files in seconds.
+- Open source and extensible via plugin system.
+
+## 🚀 Quick start
+
+1. Start with a collection of markdown files. These could be your blog posts, project docs, or any other type of content. Each file can be enhanced with a YAML frontmatter to include metadata.
```

Comment:
It would be kind of nice to just screenshot the sections from the front page here and use the same text. I think it is worth it and useful for historical purposes (one day the markdowndb front page will have changed).

So what i mean is something like:

```
### You have a folder of markdown content

For example, your blog posts. Each file can have a YAML frontmatter header with metadata like title, date, tags, etc.

{Insert screenshot i.e.}

```

<img width="857" alt="image" src="https://github.com/datopian/portaljs/assets/180658/d9db95cf-8475-4243-807b-5e46f92ccf60">

etc etc